api/routes/auth/certifications/files.endpoint.js

```diff
@@ -0,0 +1,124 @@
+const fs = require('fs');
+const FormData = require('form-data');
+
+const {
+  buildForm,
+  login,
+  unauthenticatedTest,
+  unauthorizedTest
+} = require('../../../endpoint-tests/utils');
+
+describe('auth/certifications/files endpoints', () => {
+  describe('Get a file by fileId | GET /auth/certifications/:fileID', () => {
+
+    const url = (fileID) => `/auth/certifications/files/${fileID}`;
+
+    describe('when authenticated as a user with permission', () => {
+      let api;
+
+      beforeAll(async () => {
+        api = login();
+      });
+
+      unauthenticatedTest('get', url(0));
+      unauthorizedTest('get', url(0));
+
+      it('with an invalid request', async () => {
+        const response = await api.get(
+          url(null)
+        );
+
+        expect(response.status).toEqual(400);
+      });
+      
+      it('with a valid request', async () => {
+        const response = await api.get(
+          url('74aa0d06-ae6f-472f-8999-6ca0487c494f')
+        );
+
+        expect(response.status).toEqual(200);
+        expect(response.data).toMatchSnapshot();
+      });
+    });
+  });
+
+  describe('Upload a state certification letter/file | POST /auth/certifications/files', () => {
+    const form = buildForm({ file: 'this is my file' });
+    const url = '/auth/certifications/files'
+
+    unauthenticatedTest('post', url);
+    unauthorizedTest('post', url);
+
+    describe('when authenticated as a user with permission', () => {
+      let api;
+      beforeAll(async () => {
+        api = login();
+      });
+
+      it('with a non-existant apd ID', async () => {
```

Comment:
probably not an APD id.  I also looked it up and existent is the right spelling  https://en.wiktionary.org/wiki/existant

Reply:
good catch. this was copy + pasted over from another test. Not an actual relevant test I wrote or want included so I will take it out.

Reply:
good call, thats what I get for not thoroughly reading copy + pasted code 🤣